index.js

```diff
@@ -2,8 +2,8 @@ const path = require('path')
 const fs = require('fs')
 const qs = require('querystring')
 
-// create global paramPattern but define it in the router to allow custom char for parameterized routes
-let paramPattern
+
+const paramPattern = /:([^\/%]+)/
```

Comment:
Oups I copy/pasted it without reading it
---
AFAIK there is absolutely nothing wrong with both approach and for small to medium replace cases they are both equivalent. It is a matter of preference and replace is more aligned with the current style of the plugin. It would be faster for deep trees too